enumerables.rb

```diff
@@ -0,0 +1,135 @@
+module Enumerable
+
+  def my_each
+    return to_enum unless block_given?
```

Comment:
* [ ] Let us replace ```  return to_enum unless block_given? ``` with ```return p to_enum(__method__) unless block_given?``` that way we would return enum of the method when no block is given. Take a careful look at the answers you are getting now. results changes from ```#<Enumerator: [1, 2, 3, 4, 5]:each>``` to ```#<Enumerator: [1, 2, 3, 4, 5]:my_each>```